app/scripts/controllers/permissions/permission-log.test.js

```diff
@@ -1,6 +1,5 @@
 import nanoid from 'nanoid';
```

Comment:
Please add a regression test confirming that there are specifically no `request` and `response` properties.